src/com/hms_networks/americas/sc/package.html

```diff
@@ -14,7 +14,7 @@
 This library is dependent on the HMS Networks MU Americas Solution Center library for file
 utilities, found at https://github.com/hms-networks/sc-flexy-logger-lib.
 
-@version 2.1
+@version 2.2
```

Comment:
spelling error in commit title

I assume you are planning for a release so this (https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::78561cc3-4c3b-4047-b463-167a3831b5ba?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A5%2C%5C%22sectionId%5C%22%3A19%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3Accb41d2097924ef6aa065081838e1e83%40thread.skype%22%7D&tenantId=5c7c1590-4488-4e42-bc9c-15218f8ac994) should be followed

Reply:
Fixed

Reply:
spelling error in commit title

Reply:
Still a spelling error in commit title
RELEASE: Incremented version to 2.2